swagger-spec/view_only_links/nodes.yaml

```diff
@@ -1 +1,420 @@
-# /view_only_links/{link_id}/nodes/
+get:
+  summary: List all nodes
+  description: >-
+
+    A list of nodes associated with the View Only Link.
```

Comment:
I think the description from the definition might be better here: "The list of nodes which this view only link gives read-only access to."